codex-ui/src/vue/components/context-menu/ContextMenu.vue

```diff
@@ -41,10 +48,56 @@ withDefaults(
      * Array of items for context menu
      */
     items: Item[];
+
+    /**
+     * Message what occurs as a result of the search
+     */
+    message: string;
```

Comment:
i think that is we use it just for search, it should be moved to constants as "Nothing found" or smth same